droidlet/dashboard/web/src/components/Timeline/README.md

```diff
@@ -20,7 +20,7 @@ The timeline relies on the vis.js Timeline and DataSet frameworks for the render
 
 vis.js [groups](https://visjs.github.io/vis-timeline/docs/timeline/#groups) allows event types (e.g. Perception, Dialogue, and Interpreter) to be grouped together in a separate row on the timeline.
 
-Other available [options](https://visjs.github.io/vis-timeline/docs/timeline/#Configuration_Options) allow for features such as tooltips, maximum window size, and automatic scrolling. Tooltip `template` allows for custom HTML formatting of tooltips. `zoomMax` sets the maximum zoom interval for the visible range in milliseconds (currently set to 24 hours). `rollingMode` centers the timeline until the user drags the timeline, causing a toggle button to appear on the right side of the timeline. `stack: false` prevents the events from stacking on top of each other.
```

Comment:
curios: why remove `stack: false` in code and here?

Reply:
Just for readability! So for example, in the screenshot the interpreter and dialogue events are very close together. If I disabled event stacking, you wouldn't be able to see the dialogue label at all since that box is smaller. I originally disabled event stacking because range events didn't overlap (since each box was so thin)